### PR TITLE
Autofocus with autocomplete and invert autocomplete highlighting

### PR DESCRIPTION
**Pull request by Delerme**
Give this a try before approving if you can, my dev environment is a little wonky

### CODE REVIEW

File: tests/core/models/autocompletedata.js [hunk 1]
```diff
@@ -60,15 +60,15 @@ describe('constructing from a response', () => {
         shortValue: 'Jesse Sharps',
         filter: {},
         matchedSubstrings: [ { offset: 8, length: 2 } ],
-        highlightedValue: 'Jesse Sh<strong>ar</strong>ps'
+        highlightedValue: '<strong>Jesse Sh</strong>ar<strong>ps</strong>'
```
Comment (benmcginnis) [inline tests/core/models/autocompletedata.js:63]: for these tests, which value is the input search string?

Comment (Delerme) [reply tests/core/models/autocompletedata.js:63]: It's applying the matchedSubstrings to the `value` strings above, but the matched substrings are kind of arbitrary and don't really reflect a particular query. The server takes the query and figures out the substrings, while the library doesn't care about the original query and just uses the matched substrings, so these tests similarly just make up some substrings with no query in context. 

It's a little confusing, especially with the multi substring test below, but main thing is just look at the matchedSubstrings for what should be (or now that it's inverted, should not be) highlighted

Comment (benmcginnis) [reply tests/core/models/autocompletedata.js:63]: great ok, I think we also need to change the default scss [here](https://github.com/yext/answers/blob/ed5c0c992abf496a7f8a015539fc49361dc14fb1/src/ui/sass/modules/_AutoComplete.scss#L12) to `$font-weight-normal` and [here](https://github.com/yext/answers/blob/ed5c0c992abf496a7f8a015539fc49361dc14fb1/src/ui/sass/modules/_AutoComplete.scss#L55) to `$font-weight-semibold`

Comment (Delerme) [reply tests/core/models/autocompletedata.js:63]: Done

**Review by benmcginnis — APPROVED** [review]

**Pull request merged**